integration-tests/hibernate-reactive-db2/src/main/resources/META-INF/persistence.xml

```diff
@@ -0,0 +1,40 @@
+<persistence xmlns="http://xmlns.jcp.org/xml/ns/persistence"
```

Comment:
After https://github.com/quarkusio/quarkus/issues/9859 we should be able to see these integration tests to work while not needing neither a `persistence.xml` nor a `beans.xml` , right?

Reply:
without persistence.xml yes, without beans.xml I'm not sure... it might already be possible to remove that actually, I'll check during impl of #9859

Reply:
Right, I wondered about the `beans.xml`, but preferred to merge rather than hold it. +1 to check when you look at `persistence.xml`.